src/test/java/jenkins/branch/OrganizationFolderTest.java

```diff
@@ -24,21 +24,42 @@
 
 package jenkins.branch;
 
-import hudson.model.ItemGroup;
-import hudson.model.Result;
-import hudson.model.TaskListener;
-import hudson.model.View;
-import jenkins.branch.harness.MultiBranchImpl;
-import jenkins.scm.api.SCMNavigator;
-import jenkins.scm.impl.SingleSCMNavigator;
-import hudson.scm.NullSCM;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
```

Comment:
Imports order is: static as a group after all non-static. All imports strict a-z